src/js/components/Modal.jsx

```diff
@@ -109,13 +109,13 @@ export default function Modal({ title, children, closeText, confirmText, onClose
           </ModalTitle>
           <ModalBody>{children}</ModalBody>
           <ButtonContainer>
-            <Button onClick={onClose} type="button" style={{ backgroundColor: "#6c757d" }}>
-              {closeText}
-            </Button>
+            <Button buttonText={closeText} clickHandler={onClose} secondaryButton={true} />
             {typeof onConfirm === "function" && (
-              <Button onClick={onConfirm} type="button" style={{ marginLeft: "1rem" }}>
-                {confirmText}
-              </Button>
+              <Button
+                buttonText={confirmText}
+                clickHandler={onConfirm}
+                extraStyle={{ marginLeft: "1rem" }}
```

Comment:
I think we can use the default `style` attribute here.

Reply:
I tried that but it didn't work. That's why I added the `extraStyle` prop.